src/routes.php

```diff
@@ -3,14 +3,15 @@
 use Slim\Http\Request;
 use Slim\Http\Response;
 
-
-// Routes
 $app->get('/', function (Request $request, Response $response, array $args) {
     // Sample log message
     $this->logger->info("Slim-Skeleton '/' route");
     return $this->renderer->render($response, 'index.phtml', $args);
 });
 
+// URL routes - to display pages
 $app->get('/admin', 'adminControllerFactory');
 
-// API routes:
\ No newline at end of file
+// API routes
+$app->post('/api/login', 'LoginController');
+$app->post('/api/registerUser', 'RegisterUserControllerFactory');
```

Comment:
`/api/reigsterUser` should point to `RegisterUserController` which and the key of `RegisterUserController` should point to `new \Portal\Factory\RegisterUserControllerFactory();`